examples/storybook/src/stories/QueryRuleContext.stories.ts

```diff
@@ -0,0 +1,162 @@
+          [ngStyle]="{ 'background-image': 'url(' + hit.image + ')' }"
+        ></div>
+
+        <div class="playground-hits-desc">
+          <p>
+            <ais-highlight [hit]="hit" attribute="title"></ais-highlight>
+          </p>
+          <p>Genre: {{ hit.genre.join(", ") }}</p>
+        </div>
+      </li>
+    </ol>
+  </ng-template>
+  `,
+};
+
+storiesOf('QueryRuleContext', module)
+  .addDecorator(meta)
+  .add('default', () => ({
+    component: wrapWithHits({
+      ...moviesConfig,
+      template: `
+      <ul>
+        <li>Select the "Drama" category and The Shawshank Redemption appears</li>
+        <li>Select the "Thriller" category and Pulp Fiction appears</li>
+        <li>Type <q>music</q> and a banner will appear.</li>
+      </ul>
+
+      <ais-query-rule-context
+        [trackedFilters]="trackedFilters"
+      ></ais-query-rule-context>
+
+      <ais-query-rule-custom-data>
+        <ng-template let-items="items">
+          <div *ngFor="let item of items">
+            <h2>{{ item.title }}</h2>
+            <a [href]="item.link">
+              <img
+                [src]="item.banner"
+                [alt]="item.title"
+                [style.width]="'100%'"
+              />
+            </a>
+          </div>
+        </ng-template>
+      </ais-query-rule-custom-data>
+      `,
+      methods: {
+        trackedFilters: {
+          genre: () => ['Thriller', 'Drama'],
+        },
+      },
+    }),
+  }))
+  .add('with initial filter', () => ({
+    component: wrapWithHits({
+      ...moviesConfig,
+      template: `
+    <ul>
+      <li>
+        Select the "Drama" category and The Shawshank Redemption appears
+      </li>
+      <li>Select the "Thriller" category and Pulp Fiction appears</li>
+      <li>Type <q>music</q> and a banner will appear.</li>
+    </ul>
+
+    <ais-configure
+      [searchParameters]="{
+        disjunctiveFacetsRefinements: {
+          genre: ['Drama']
+        }
+      }"
+    ></ais-configure>
+
+    <ais-query-rule-context
+      [trackedFilters]="trackedFilters"
+    ></ais-query-rule-context>
+
+    <ais-query-rule-custom-data>
+      <ng-template let-items="items">
+        <div *ngFor="let item of items">
+          <h2>{{ item.title }}</h2>
+          <a [href]="item.link">
+            <img
+              [src]="item.banner"
+              [alt]="item.title"
+              [style.width]="'100%'"
+            />
+          </a>
+        </div>
+      </ng-template>
+    </ais-query-rule-custom-data>
+    `,
+      methods: {
+        trackedFilters: {
+          genre: () => ['Thriller', 'Drama'],
+        },
+      },
+    }),
+  }))
+  .add('with initial rule context', () => ({
+    component: wrapWithHits({
+      ...moviesConfig,
+      template: `
+    <ul>
+      <li>
+        Select the "Drama" category and The Shawshank Redemption appears
+      </li>
+      <li>Select the "Thriller" category and Pulp Fiction does not appear</li>
```

Comment:
It does appear with the category selected.

Reply:
Oh that’s the story Francois changed its usage of

Reply:
Oops, I forgot to change the description.